server/protocols/jmap-integration-testing/memory-jmap-integration-testing/pom.xml

```diff
@@ -169,6 +169,7 @@
                 <configuration>
                     <reuseForks>false</reuseForks>
                     <forkCount>1C</forkCount>
+                    <excludedGroups combine.self="override">org.apache.james.jmap.categories.CassandraAndElasticSearchOnly</excludedGroups>
```

Comment:
cannot we consider that every test should run on memory (maybe just need to override + ignore one of them)?

the CassandraAndElasticSearchOnly category will then be used later when adding the swift integration tests

Reply:
100% agree